Content/GUI/Browser.cs

```diff
@@ -318,16 +321,33 @@ public override void Draw(SpriteBatch spriteBatch)
 			{
 				GUIHelper.DrawOutline(spriteBatch, GetDimensions().ToRectangle(), ModContent.GetInstance<GUIConfig>().buttonColor.InvertColor());
 				HandleText();
+
+				// draw ime panel, note that if there's no composition string then it won't draw anything
+				Main.instance.DrawWindowsIMEPanel(GetDimensions().Position());

```

Comment:
Potential future concern: do we perhaps want to have our own IME panel that respects theme options in the mod config? (color, box style, etc)

Reply:
More specifically, it might be better to detour the drawing logic and apply our theme changes accordingly when the context is within a DragonLens UI. Depends on how easy that ends up being.